api/src/main/java/io/strimzi/api/kafka/model/AclRule.java

```diff
@@ -82,7 +94,8 @@ public void setHost(String host) {
 
     @Description("Operation which will be allowed or denied. " +
             "Supported operations are: Read, Write, Create, Delete, Alter, Describe, ClusterAction, AlterConfigs, DescribeConfigs, IdempotentWrite and All.")
-    @JsonProperty(required = true)
+    @DeprecatedProperty(description = "This property is replaced by the `operations` property")
```

Comment:
The deprecated property annotation has fields `movedToPath` which you should use. It will generate the docs for the move.